src/main/java/io/reactivex/CompletableObserver.java

```diff
@@ -17,7 +17,36 @@
 import io.reactivex.disposables.Disposable;
 
 /**
- * Represents the subscription API callbacks when subscribing to a Completable instance.
+ * Provides a mechanism for receiving push-based notification of a valueless completion or an error.
+ * <p>
+ * When a {@code CompletableObserver} is subscribed to a {@link CompletableSource} through the {@link CompletableSource#subscribe(CompletableObserver)} method,
+ * the {@code CompletableSource} calls {@link #onSubscribe(Disposable)}  with a {@link Disposable} that allows
+ * cancelling the sequence at any time. A well-behaved
```

Comment:
should we use `cancelling` here or `disposing` since the method is called dispose?